qcodes/utils/validators.py

```diff
@@ -12,6 +12,7 @@
 from typing import Dict as TDict
 from typing import Generic, Hashable
 from typing import List as TList
+from typing import Literal
```

Comment:
We need to import Literal from typing_extensions for python 3.7 support until we drop support for 3.7

Reply:
Fixed!